float32[] goal
---
trajectory_msgs/JointTrajectory path
string message